packages/block-library/src/latest-posts/index.php

```diff
@@ -37,7 +37,7 @@ function render_block_core_latest_posts( $attributes ) {
 		$list_items_markup .= sprintf(
 			'<li><a href="%1$s">%2$s</a>',
 			esc_url( get_permalink( $post ) ),
-			esc_html( $title )
+			$title
```

Comment:
Is it safe? This is also used to render it on the frontend.

Reply:
> Is it safe? This is also used to render it on the frontend.

I think it's good to be prudent of, but in this instance I believe it's okay, given the following security consideration notes:

https://codex.wordpress.org/Function_Reference/the_title#Security_considerations

Reply:
Thanks for sharing. That makes this code completely valid.